### PR TITLE
chore(pickple): Cleanup more bad auditlog pickled data

### PR DESCRIPTION
**Pull request by evanpurkhiser**
This is a follow up to https://github.com/getsentry/sentry/pull/17545.

There were more badly serialized Team objects in the auditlog that were not covered by Dan's previous migration.

This handles ALL auditlog entries

### CODE REVIEW

File: src/sentry/migrations/0090_fix_auditlog_pickled_data_take_2.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+from django.db import migrations
+
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def cleanup_audit_log_data(apps, schema_editor):
+    """
+    Fix `AuditLogEntry` rows that have pickled `Team` models in their `data` field. Also
+    fixes some rows where flags were serialized as a BitHandler rather than a int.
```
Comment (wedamija) [inline src/sentry/migrations/0090_fix_auditlog_pickled_data_take_2.py:13]: Nit: Remove the  `BitHandler` stuff from the comment

**Review by wedamija — COMMENTED** [review]
You should generate your migration with `sentry django makemigrations -n fix_auditlog_pickled_data_take_2 --empty` so that it updates the lockfile. Or you can just manually update the lockfile yourself.

**Comment by EvanPurkhiser** [comment]
> You should generate your migration with sentry django makemigrations -n fix_auditlog_pickled_data_take_2 --empty so that it updates the lockfile. Or you can just manually update the lockfile yourself.

I actually did do this, just forgot to commit the lockfile :)

It's here now.

**Review by wedamija — APPROVED** [review]
lgtm

**Pull request merged**